bin/check-rabbitmq-queue.rb

```diff
@@ -50,9 +51,9 @@ class CheckRabbitMQMessages < Sensu::Plugin::Check::CLI
          boolean: true,
          default: false
 
-  option :user,
+  option :username,
```

Comment:
I like making it consistent but this makes it a breaking change. This will need to be called out in the CHANGELOG.